wire-ios-data-model/Tests/Source/Model/Messages/ZMClientMessageTests+OTR_Legacy.swift

```diff
@@ -0,0 +1,486 @@
+//
+// Wire
+// Copyright (C) 2016 Wire Swiss GmbH
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+// 
+
+import XCTest
+@testable import WireDataModel
+
+final class ClientMessageTests_OTR_Legacy: BaseZMClientMessageTests {}
+
+// MARK: - Payload creation
+extension ClientMessageTests_OTR_Legacy {
```

Comment:
These tests are what was already existing, I just renamed it include the word "Legacy", we'll delete all these tests later when we remove Cryptobox.